common/lib/xmodule/xmodule/js/src/capa/display.js

```diff
@@ -722,7 +713,7 @@
                         display = that.inputtypeDisplays[$(inputtype).attr('id')];
                         showMethod = that.inputtypeShowAnswerMethods[cls];
                         if (showMethod != null) {
-                            results.push(showMethod(inputtype, display, answers));
+                            results.push(showMethod(inputtype, display, answers, response.correct_status_html));
```

Comment:
This is where rendering of show answer actually happens (via a specific inputtype function).